<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Export2KMLDialog</class>
  <widget class="QDialog" name="Export2KMLDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Export 2KML</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <!-- Table of layers -->
      <item>
        <widget class="QTableWidget" name="tableLayers">
          <property name="columnCount">
            <number>0</number>
          </property>
          <property name="rowCount">
            <number>0</number>
          </property>
        </widget>
      </item>
      <!-- KML/KMZ radio buttons -->
      <item>
        <layout class="QHBoxLayout" name="horizontalLayoutFormat">
          <item>
            <widget class="QRadioButton" name="radioKML">
              <property name="text">
                <string>KML</string>
              </property>
              <property name="checked">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QRadioButton" name="radioKMZ">
              <property name="text">
                <string>KMZ</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <!-- Output path with browse button -->
      <item>
        <layout class="QHBoxLayout" name="horizontalLayoutOutput">
          <item>
            <widget class="QLineEdit" name="editOutput"/>
          </item>
          <item>
            <widget class="QPushButton" name="btnSelectOutput">
              <property name="text">
                <string>Browse Output</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <!-- Progress Bar (hidden by default) -->
      <item>
        <widget class="QProgressBar" name="progressBar">
          <property name="minimum">
            <number>0</number>
          </property>
          <property name="maximum">
            <number>100</number>
          </property>
          <property name="value">
            <number>0</number>
          </property>
          <property name="visible">
            <bool>false</bool>
          </property>
          <property name="textVisible">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <!-- Bottom buttons: Browse Input, Run, Cancel -->
      <item>
        <layout class="QHBoxLayout" name="horizontalLayoutButtons">
          <item>
            <widget class="QPushButton" name="btnBrowseFiles">
              <property name="text">
                <string>Browse Input Files</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="btnRun">
              <property name="text">
                <string>Run</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="btnCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
